client/app/cloud/project/compute/infrastructure/region.service.js

```diff
@@ -27,6 +27,12 @@ class CloudRegionService {
       delete region.disabled; // eslint-disable-line
     }
   }
+
+  static checkAvailability(region) {
+    if (region.notAvailable) {
+      _.set(region, 'disabled', 'IN_ACTIVE');
```

Comment:
legacy code has used text message in region.disabled. Its mentioned in a lot of places. It will take a long time and testing effort to change it now. I have changed the method name and removed side effects as mentioned.